### PR TITLE
[ja][4.x]4-4-migration-guide update

### PR DESCRIPTION
**Pull request by yKanazawa**
https://book.cakephp.org/4/en/appendices/4-4-migration-guide.html
https://book.cakephp.org/4/ja/appendices/4-4-migration-guide.html

https://book.cakephp.org/4/en/appendices/migration-guides.html
https://book.cakephp.org/4/ja/appendices/migration-guides.html

### CODE REVIEW

File: ja/appendices/4-1-migration-guide.rst [hunk 1]
```diff
@@ -26,9 +26,9 @@ CakePHP 4.1.0にアップグレードするには、次の Composer コマンド
 Controller
 ----------
 
-* ``PaginatorComponent`` の ``sortWhitelist`` オプションは非推奨は非推奨になりました。
+* ``PaginatorComponent`` の ``sortWhitelist`` オプションは非推奨になりました。
```
Comment (yKanazawa) [inline ja/appendices/4-1-migration-guide.rst:29]: fix typo

File: ja/appendices/4-1-migration-guide.rst [hunk 2]
```diff
@@ -26,9 +26,9 @@ CakePHP 4.1.0にアップグレードするには、次の Composer コマンド
...
   代わりに、``sortableFields`` を使用してください。
-* ``PaginatorComponent`` の ``whitelist`` オプションは非推奨は非推奨になりました。
+* ``PaginatorComponent`` の ``whitelist`` オプションは非推奨になりました。
```
Comment (yKanazawa) [inline ja/appendices/4-1-migration-guide.rst:31]: fix typo

**Comment by othercorey** [comment]
The 4.4 migration notes reference missing sections.

> /home/runner/work/docs/docs/ja/appendices/4-4-migration-guide.rst:74: WARNING: undefined label: controller-viewclasses
/home/runner/work/docs/docs/ja/appendices/4-4-migration-guide.rst:77: WARNING: unknown document: /controllers/components/check-http-cache
/home/runner/work/docs/docs/ja/appendices/4-4-migration-guide.rst:78: WARNING: undefined label: controller-viewclasses
/home/runner/work/docs/docs/ja/appendices/4-4-migration-guide.rst:160: WARNING: undefined label: console-command-description

**Comment by othercorey** [comment]
Looks good!

**Pull request merged**